xl2times/__main__.py

```diff
@@ -95,19 +98,18 @@ def convert_xl_to_times(
         transforms.convert_aliases,
         transforms.rename_cgs,
         transforms.fix_topology,
-        transforms.final_cleanup,
         transforms.convert_to_string,
-        lambda config, tables: dump_tables(
+        lambda config, tables, model: dump_tables(
             tables, os.path.join(output_dir, "merged_tables.txt")
         ),
-        lambda config, tables: produce_times_tables(config, tables),
+        lambda config, tables, model: produce_times_tables(config, tables),
     ]
 
     input = raw_tables
     output = {}
     for transform in transform_list:
         start_time = time.time()
-        output = transform(config, input)
+        output = transform(config, input, model)
```

Comment:
So it looks like with the `TimesModel` class we're switching away from a functional style transformation (each loop iteration is `output = f(input)`) to a more imperative style (`model` is modified in-place). Both styles have their merits, so it probably doesn't matter which style we pick as long as we're consistent.

Is the intention to slowly migrate everything in `input` to `model` and switch from functional style to imperative style eventually?

Reply:
With this we are also getting closer to addressing #11. :-) Not sure about moving completely away from the functional style... I see `TimesModel` class as a great separator between a model specification approach (e.g. Veda) and how the tool itself treats a TIMES model and what it can then do with it.  But let's discuss this further.